src/net.h

```diff
@@ -460,6 +460,8 @@ class CConnman
     Mutex mutexMsgProc;
     std::atomic<bool> flagInterruptMsgProc{false};
 
+    CThreadInterrupt g_interrupt_dnsseed_thread;
+    CThreadInterrupt g_interrupt_addcon_thread;
```

Comment:
```suggestion
    CThreadInterrupt m_interrupt_addcon_thread;
```

Looks like a member to me, no?

Reply:
Sure! Thank you. [Updated](https://github.com/bitcoin/bitcoin/pull/19029#issuecomment-631595378).